### PR TITLE
Gyak5

### PR DESCRIPTION
**Pull request by holyburger**
3.3c

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 1]
```diff
@@ -38,8 +38,7 @@ public boolean fireTorpedo(FiringMode firingMode) {
 
     boolean firingSuccess = false;
 
-    switch (firingMode) {
-      case SINGLE:
+    if (firingMode==FiringMode.SINGLE) {
```
Comment (Hbrndzs) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java:41]: I'd do the same

File: src/main/java/hu/bme/mit/spaceship/TorpedoStore.java [hunk 2]
```diff
@@ -7,7 +7,7 @@
 */
 public class TorpedoStore {
 
-  private double FAILURE_RATE = 0.0;
+  private double failureRate = 0.0;
```
Comment (Hbrndzs) [inline src/main/java/hu/bme/mit/spaceship/TorpedoStore.java]: Rename it

**Review by Hbrndzs — CHANGES_REQUESTED** [review]
Not bad

**Review by Hbrndzs — APPROVED** [review]
Nice

File: src/main/java/hu/bme/mit/spaceship/GT4500.java (hunk 1 above)
Comment (hemcraft) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java:41]: megfelel

**Pull request merged**